### PR TITLE
removed  REG_SZ  added  REG_DWORD

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report #8526 , i  removed **REG_SZ**   and added   **REG_DWORD**

https://user-images.githubusercontent.com/3296790/97083291-8a530200-162c-11eb-83e6-a4cc001a18d5.JPG

### CODE REVIEW

**Review by illfated — APPROVED** [review]
Looks valid to me.

**Comment by nam31** [comment]
@linque1 PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 1]
```diff
@@ -1659,7 +1659,7 @@ You can turn off **Enhanced Notifications** as follows:
 
   -or-
 
-- Create a new REG_SZ registry setting named **DisableEnhancedNotifications** in **HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows Defender\Reporting** to a value of **1**.
+- Create a new REG_DWORD registry setting named **DisableEnhancedNotifications** in **HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows Defender\Reporting** to a value of **1**.
```
Comment (illfated) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:1662]: The phrasing in this sentence seems to be made for a case where an existing key should be set to a specific value. When creating a new key, I am fairly certain we should phrase it like this:
```suggestion
- Create a new REG_DWORD registry setting named **DisableEnhancedNotifications** in **HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows Defender\Reporting** and enter the decimal value **1**.
```
( I know, you are not the original author of this sentence, but we aim to improve the documentation whenever we can. )

**Comment by illfated** [comment]
Sorry for the late change request. The last couple of weeks have been rough, so I have not always performed as well as I hoped. We may also want to have an additional phrasing check by JohanFreelancer9, if possible.

**Review by illfated — APPROVED** [review]
#sign-off

**Comment by illfated** [comment]
Metadata section from the GH doc page:

```
---
title: Manage connections from Windows 10 operating system components to Microsoft services
description: Learn how to minimize connections from Windows to Microsoft services, and configure particular privacy settings related to these connections.
ms.assetid: ACCEB0DD-BC6F-41B1-B359-140B242183D9
ms.reviewer: 
keywords: privacy, manage connections to Microsoft, Windows 10, Windows Server 2016
ms.prod: w10
ms.mktglfcycl: manage
ms.sitesec: library
ms.localizationpriority: high
audience: ITPro
author: linque1
ms.author: obezeajo
manager: robsize
ms.collection: M365-security-compliance
ms.topic: article
ms.date: 7/7/2020
---
```

Not sure if there are other useful reviewers to be picked from that list.

**Pull request merged**